calibration/experiments/gcm_driven_scm/run_calibration.jl

```diff
@@ -17,39 +15,44 @@ experiment_dir = dirname(Base.active_project())
 const model_interface = joinpath(experiment_dir, "model_interface.jl")
 const experiment_config =
     YAML.load_file(joinpath(experiment_dir, "experiment_config.yml"))
-const n_iterations = experiment_config["n_iterations"]
-const ensemble_size = experiment_config["ensemble_size"]
-const output_dir = experiment_config["output_dir"]
-const model_config = experiment_config["model_config"]
-const z_max = experiment_config["z_max"]
-const eki_timestep = experiment_config["eki_timestep"]
-const const_noise = experiment_config["const_noise"]
-const prior =
-    CAL.get_prior(joinpath(experiment_dir, experiment_config["prior"]))
+
+# unpack experiment_config vars into scope 
+for (key, value) in experiment_config
+    @eval const $(Symbol(key)) = $value
```

Comment:
Usually this is not a great idea to do eval of something read from file, is this how it is typically done at the moment?

Reply:
It's just done in this file since a lot of the configuration variables are used. I had them listed out before like 
```
const n_iterations = experiment_config["n_iterations"]
const ensemble_size = experiment_config["ensemble_size"]
...
```
I can add that back if it's safer.